tree/tree/src/InternalTreeUtils.cxx

```diff
@@ -200,6 +202,9 @@ ROOT::TreeUtils::RFriendInfo GetFriendInfo(const TTree &tree, bool retrieveEntri
       const auto *alias_c = tree.GetFriendAlias(frTree);
       const std::string alias = alias_c != nullptr ? alias_c : "";
 
+      auto *treeIndex = frTree->GetTreeIndex();
+      treeIndexes.emplace_back(static_cast<TVirtualIndex *>(treeIndex ? treeIndex->Clone() : nullptr));
```

Comment:
Right so now we `Clone` irrespectively of being in local/distributed mode. It's an extra operation but allows for the distributed case